### PR TITLE
feat(url): Use path instead of url if available

### PR DESCRIPTION
**Pull request by Crocmagnon**
Fixes "RemovedInDjango40Warning: django.conf.urls.url() is deprecated in favor of django.urls.re_path()."

### CODE REVIEW

**Comment by Crocmagnon** [comment]
Some tests fail but the error doesn't seem related to the changes.

File: advanced_filters/urls.py [hunk 1]
```diff
@@ -1,14 +1,30 @@
-from django.conf.urls import url
+try:
+    from django.urls import path
+except ImportError:
+    path = None
+    from django.conf.urls import url
```
Comment (ulgens) [inline advanced_filters/urls.py:5]: `path` function came with Django 2.2, which is the current minimum supported version. No need for the if block.

**Review by ulgens — CHANGES_REQUESTED** [review]

File: advanced_filters/urls.py (hunk 1 above)
Comment (Crocmagnon) [reply advanced_filters/urls.py:5]: This PR is not relevant anymore, the develop branch already includes this fix.

**Comment by Crocmagnon** [comment]
Not relevant anymore.

**Pull request closed without merging**